apps/services/endorsements/api/src/app/modules/endorsementList/endorsementList.service.ts

```diff
@@ -232,24 +239,180 @@ export class EndorsementListService {
+      )
+      .moveDown()
+
+      .font(fontBold)
+      .text('Fjöldi skráðir: ')
+      .font(fontRegular)
+      .text(endorsementList.endorsements.length)
+      .moveDown(2)
+
+    if (endorsementList.endorsements.length) {
+      doc.fontSize(big).text('Yfirlit meðmæla').fontSize(regular).moveDown()
+      for (const val of endorsementList.endorsements) {
+        doc.text(
+          val.created.toLocaleDateString(locale) +
+            ' ' +
+            (val.meta.fullName ? val.meta.fullName : 'Nafn ótilgreint'),
+        )
+      }
+    }
+    doc
+      .moveDown()
+
+      .fontSize(regular)
+      .text(
+        'Þetta skjal var framkallað sjálfvirkt þann: ' +
+          new Date().toLocaleDateString(locale),
+      )
+    doc.end()
+    return await getStream.buffer(doc)
+  }
+
+  async emailPDF(
+    listId: string,
+    recipientEmail: string,
+  ): Promise<{ success: boolean }> {
     const endorsementList = await this.endorsementListModel.findOne({
-      where: {
-        id: listId,
-      },
+      where: { id: listId },
+      include: [
+        {
+          model: Endorsement,
+        },
+      ],
     })
     if (!endorsementList) {
       throw new NotFoundException(['This endorsement list does not exist.'])
     }
+    let ownerName = await this.getOwnerInfo(
+      endorsementList?.id,
+      endorsementList.owner,
+    )
+    this.logger.info(
+      `sending list ${listId} to ${recipientEmail} from ${environment.email.sender}`,
+    )
     try {
-      return (await this.nationalRegistryApi.getUser(endorsementList.owner))
-        .Fulltnafn
-    } catch (e) {
-      return ''
+      await this.emailService.sendEmail({
+        from: {
+          name: environment.email.sender,
+          address: environment.email.address,
+        },
+        to: [
+          {
+            // message can be sent to any email so recipient name is unknown
+            name: recipientEmail,
+            address: recipientEmail,
+          },
+        ],
+        subject: `Meðmælendalisti "${endorsementList?.title}"`,
+        template: {
+          title: `Meðmælendalisti "${endorsementList?.title}"`,
+          body: [
+            {
+              component: 'Heading',
+              context: {
+                copy: `Meðmælendalisti "${endorsementList?.title}"`,
+              },
+            },
+            { component: 'Copy', context: { copy: 'Sæl/l/t' } },
+            {
+              component: 'Copy',
+              context: {
+                copy: `Meðfylgjandi er meðmælendalisti "${endorsementList?.title}", 
+                sem ${ownerName} er skráður ábyrgðarmaður fyrir. 
+                Vakin er athygli á lögum um persónuvernd og vinnslu persónuupplýsinga nr. 90/2018.`,
+              },
+            },
+            { component: 'Copy', context: { copy: 'Kær kveðja,' } },
+            { component: 'Copy', context: { copy: 'Ísland.is' } },
+          ],
+        },
+        attachments: [
+          {
```

Comment:
can you comment out the single object in this array just while we are establishing if  email sending works, once that is confirmed we can put it back in